### DISCUSSION TITLE
Workstation and the N-variants .

### DISCUSSION THREAD

**Discussion by arielsil** [AveYo/MediaCreationTool.bat#185] [Q&A]
in the 11MAY22 version, lines 1009...1911 make references to the Workstation, and to the N variants.   Could you please explain how to get these downloaded, etc?

**Comment by AveYo** [comment]
These are virtual editions, without a standalone esd image made public by microsoft.

You can only install them when doing upgrades, by renaming the script as `professionalworkstation mediacreationtool.bat` / `professionalworkstationN mediacreationtool.bat`. 

Then picking Auto Upgrade preset or running auto.bat included in iso / usb will use the overridden edition.
